### DISCUSSION TITLE
Blazor Hero other samples

### DISCUSSION THREAD

**Discussion by developer9969** [fullstackhero/blazor-starter-kit#402] [General]
Hi
I mainly develop mobile apps and web api but I am learning blazor. I find this project fantastic, but I am finding hard to follow , I mean how to customize to fit my purposes.

I.E  how to add my own splash screen
      Use Azure B2C 
      Make it run on server side not as Wasm

As anybody got any sample that uses blazor hero as serverside also that has been changed.

or a blog /video how to customize it 

Hope makes sense

**Comment by jeremiedevos** [comment]
Hi @developer9969,

You can easily change the splash screen in the index.html file.
There you can change the SVG and style that are located within the body tag.

Regarding the conversion to server side, I have no idea on how you could do that. You might take a look online on how to convert it yourself, but i for see that configuring specific elements might be a challenge, like local storage, but I have not tried it myself.

Hope this helped a bit.

**Reply by developer9969** [reply]
Thank you that will be a good start.For security reasons we cannot have it running on the browser. 

Are there samples on github using blazorhero?
I would it find it easier to see what people did so that I can customize mine

Thanks again

**Reply by jeremiedevos** [reply]
What part cannot be ran in the browser? because even if you convert it to Blazor server, it will run in your browser over a signal R. If it is the logic and data access that is the problem that will be handled by the API and will be done on the server not client.

 Regarding examples, maybe there are some in the forks: https://github.com/blazorhero/CleanArchitecture/network/members

**Comment by developer9969** [comment]
Hi 
Is more my lack of knowledge on blazor.watching few videos they all say if you have sentive data use ServerSide!!!  As data dll can be easily decompiled as its all locally downloaded. Isn't it the case???

Learning about it.would be great if you could easily switch at runtime

**Reply by jeremiedevos** [reply]
Yes, server side runs everything on the server and is in that way, more save. When using WebAssembly, everything for the front end will be downloaded and can be decompiled. However, in this template a API is setup that runs on the server. You just need to make sure that all your logic and data manipulations is being handled on the API side. 

There are a lot of explanations in the form of articles and videos that can show the difference and what is saver in what situation. So if you have clear what the project needs and what features are needed, you should be able to make a good choice. On small thing, is it not always good to have a API for future proving?
